logic/main.css

```diff
@@ -484,6 +485,26 @@ input:checked + .slider:before {
 	vertical-align: top;
 }
 
+#KeyQ, #KeyA, #KeyZ, #KeyP, #Semicolon, #Slash {
+	background-color: #80c4c4;
+}
+
+#KeyW, #KeyS, #KeyX, #KeyO, #KeyL, #Period {
+	background-color: #ca80ca;
+}
+
+#KeyE, #KeyD, #KeyC, #KeyI, #KeyK, #Comma {
+	background-color: #80b780;
+}
+
+#KeyR, #KeyF, #KeyV, #KeyT, #KeyG, #KeyB {
+	background-color: #7a93c0;
+}
+
+#KeyY, #KeyH, #KeyN, #KeyU, #KeyJ, #KeyM {
+	background-color: #8080ca;
+}
+
```

Comment:
I'm thinking, so you want a certain colour for the proper fingering right, and then when active for that level, it would change colour?

If we created classes like `finger-01` (through to 10) and assigned them in the layoutDivs file, then we could colour them according to `.finger-01` and `.finger-01.active` right?

Especially if you just want it to be darker, and then when active become light, you could actually just do

```
.finger-01 {
    background-color: somecolour;
    color: black;
    opacity: 0.5;
}

.active {
    color: white;
    opacity: 1;
}
```

right?

Reply:
That is essentially the goal, yes. That system should also make it easy to change at runtime for layouts that require different fingering like DH.

On a related note, I'd also like to be able to highlight/brighten the keys that are currently being learned (which is currently working now, albeit with a not-so-clean implementation).